src/james.data.clss.pas

```diff
@@ -207,13 +207,16 @@ function TDataStream.Save(const FileName: string): IDataStream;
 end;
 
 function TDataStream.AsString: string;
+var
+  Buf: AnsiString;
 begin
   with FStream do
   begin
-    SetLength(Result, Size);
+    SetLength(Buf, Size);
     Position := 0;
-    ReadBuffer(Pointer(Result)^, Size);
+    ReadBuffer(Pointer(Buf)^, Size);
```

Comment:
The constructor, which demands a default string in FPC and none in Delphi, and CopyFrom in FPC which should be equivalent to LoadFromStream in Delphi. CopyFrom does require the size, which LoadFromStream doesn't, but shouldn't be a problem.